docs/guide/provider-migration.md

```diff
@@ -73,10 +68,11 @@ For installation and usage instructions, please see [npm](https://npmjs.com/pack
 
 ### Handling `eth_chainId` Return Values
 
-Due to a long-standing bug, MetaMask's implementation of the `eth_chainId` RPC method has returned 0-padded values for the [default Ethereum chains](./ethereum-provider.html#chain-ids) _except_ Kovan.
-For example, instead of `0x1` and `0x2`, we currently return `0x01` and `0x02`.
+Due to a long-standing bug, MetaMask's implementation of the `eth_chainId` RPC method returned 0-padded values for the [default Ethereum chains](./ethereum-provider.html#chain-ids) _except_ Kovan.
```

Comment:
This seems to imply that we still do this? Perhaps I'm missing something.

Reply:
Addressed in: 1065b9e